TodoListClient/Controllers/TodoListController.cs

```diff
@@ -46,7 +46,7 @@ public ActionResult Index()
             //reset session on every entry to TODO's list
             TodoSessionState(SessionAction.Set);
 
-            return View(_commonDBContext.Todo.ToList());
+            return View(_commonDBContext.Todo.Where(l => l.Owner.Equals(HttpContext.User.Identity.Name)).ToList());
```

Comment:
Filtering is authorization.
It should not be done with the Name claim. but with the oid+tid. Please use [HttpContext.User.GetMsalAccountId()](https://github.com/AzureAD/microsoft-identity-web/blob/d26e79ddf9fdd2e1fa740711ec652cb2a221fc1f/src/Microsoft.Identity.Web.TokenCache/ClaimsPrincipalExtensions.cs#L80)

Reply:
@jmprieur I'm learning new things today. Thanks for input